setup.py

```diff
@@ -40,6 +40,7 @@
         'pymongo>=3.10,<4',
         'dnspython>=1.16,<2',
         'pyyaml>=5,<6',
+        'tabulate>=0.8,<0.9',
```

Comment:
What feature does this library provide? Do we need it?

Reply:
It provides a quick way to convert a list of lists into a textual table of various styles. In our case, we want the rst format to be able to include the resulting table into docs. While the docs part is not yet done, take a look at the `docgen.py` file for a closer look.
To see it in action install the package and do `astrolabe info environment-variables` or ` astrolabe info default-values`.